README.md

```diff
@@ -79,7 +79,6 @@ _Tip: Consider using the faster and smaller ES6 build if targetting a modern env
     -   LearnCode.academy MobX tutorial [Part I: MobX + React is AWESOME (7m)](https://www.youtube.com/watch?v=_q50BXqkAfI) [Part II: Computed Values and Nested/Referenced Observables (12m.)](https://www.youtube.com/watch?v=nYvNqKrl69s)
     -   [Screencast: intro to MobX](https://www.youtube.com/watch?v=K8dr8BMU7-8) - 8m
     -   [Talk: State Management Is Easy, React Amsterdam 2016 conf](https://www.youtube.com/watch?v=ApmSsu3qnf0&feature=youtu.be) ([slides](https://speakerdeck.com/mweststrate/state-management-is-easy-introduction-to-mobx))
--   [Boilerplates and related projects](http://mobxjs.github.io/mobx/faq/boilerplates.html)
```

Comment:
Let's change to https://github.com/mobxjs/awesome-mobx#boilerplates instead of just deleting. Similar for other links. Better to be specific imo.

Reply:
Should i do the same for the `[Related projects]` and `[Tutorials, Blogs & Videos]`, which i had also deleted
and if so should i split `[Tutorials, Blogs & Videos]` in three parts :  `[Tutorials]`, `[Blogs]`, `[Videos]` as each has it's own section in the [Awesome Mobx](https://github.com/mobxjs/awesome-mobx) repo ?

Reply:
Something like that, yes, please. Better than just deleting those links completely imo.